src/docs/ganache/corda/linking-a-corda-project.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Ganache | Linking a CorDapp
+layout: docs.hbs
+---
+# Linking a CorDapp Project
+
+To link a project, enter the settings by clicking the gear icon in the upper right.
+
+![Settings Icon](/img/docs/ganache/corda/corda-settings-icon.png)
+
+You should be seeing the `WORKSPACE` settings pane; if not, you can get there by clicking the `WORKSPACE` tab in the top left.
+
+![Workspaces Settings Pane Tab](/img/docs/ganache/corda/corda-workspaces-pane-tab.png)
+
+From here, there is a section labeled `PROJECTS`. Beneath this box, click the button `ADD PROJECT`. A folder selection modal will appear. Select the folder of your CordApp project. Ganache will inspect the directory, and for folders with a `gradle.build` file containing the string `"corda"`, will automatically watch the `./build/libs` folder for new/changed `.jar` files. If the folder you choose does _not_ have a matching `gradle.build` Ganache will watch the folder itself for new/changed `.jar` files.
```

Comment:
`and for folders` typo.  My guess is we can just drop the `, and`.

`containing the string `"corda"``  I wonder if it would be better to just say something along the lines of a `"valid" build.gradle` or something?

Reply:
Also, these should be `build.gradle`

Reply:
Since it can be a "valid" build.gradle, but not compatible with Ganache, i didn't want to use the word "valid" here.